app/infra/utils.js

```diff
@@ -57,4 +57,14 @@ const formatSmidge = (amount: number, separateResult) => {
   return separateResult ? { value: res.value, unit: res.unit } : `${res.value} ${res.unit}`;
 };
 
-export { formatSmidge, createError, getAbbreviatedText, getFormattedTimestamp, getAddress, formatBytes };
+const getNodeStatusText = (status) => {
```

Comment:
move it to Network.js as this is the single usage case

Reply:
fix